NasBench 1shot1

PhMueller opened the pull request.

aaronkl commented on hpolib/benchmarks/surrogates/cnn.py:
```diff
@@ -48,9 +45,16 @@ def get_configuration_space():
+                           [0, 1],  # n_units_1,  [2**4, 2**8]
+                           [0, 1],  # n_units_2,  [2**4, 2**8]
+                           [0, 1]],  # n_units_3, [2**4, 2**8]
+                'references': ["@article{klein-corr16,"
```
that is actually the wrong reference the correct one is:
@InProceedings{klein-iclr17,
 author    = {A. Klein and S. Falkner and J. T. Springenberg and F. Hutter},
 title     = {Learning Curve Prediction with {Bayesian} Neural Networks},
 booktitle = {International Conference on Learning Representations (ICLR) 2017 Conference Track},
 year      = {2017},
 month 	= apr
}

KEggensperger replied:
Agreed and fixed

aaronkl commented on hpolib/benchmarks/surrogates/svm.py:
```diff
@@ -47,7 +46,7 @@ def get_configuration_space():
-                'bounds': [[-10, 10],  # C
+                'bounds': [[-10, 10],   # C
                            [-10, 10]],  # gamma
                 'references': ["@article{klein-corr16,"
```
@INPROCEEDINGS{klein-ejs17,
 author    = {A. Klein and S. Falkner and S. Bartels and P. Hennig and F. Hutter},
 title     = {Fast Bayesian hyperparameter optimization on large datasets},
 booktitle = {Electronic Journal of Statistics},
 year      = {2017},
 volume    = {11}
}

KEggensperger replied:
Agreed and done

Commit:
init new version (#63)

PhMueller committed:
Add nasbench_1shot1 benchmark

PhMueller committed:
[skip travis] Initial Commit Nas1shot1 Benchmark

PhMueller committed:
[skip travis] Nasbench1shot1: Fix Json Serialization + Encoder

- The ConfigSpace cannot serialize hyperparameter which consists of Tuple(tuple(Any)).
Therefore, I cast them to strings and recast them to tuple later.

- Small improvements in codestyle.

- Also, I have added a JSON Encoder for handling tuple, numpy-types and others.
This is not used atm but will be then integrated in the next big pull request.

PhMueller committed:
[skip travis] Pycodestyle

PhMueller committed:
Add Recipe 1shot1 + (info about incumbents for nasbench_201.py)

PhMueller committed:
[skip travis] Improve changelog.md
